### ISSUE TITLE
Remove emphasis on first paragraph of product description

### ISSUE THREAD

**Issue by matt-downs** [NetoECommerce/Skeletal#329]
The first paragraph of the product description is emphasised as seen below:
![image](https://user-images.githubusercontent.com/6867163/35599744-01a9dc94-0676-11e8-8c1a-1cdca9c6c741.png)

As this is defined in `app.css`, this styling applies to all themes unless overwritten. IMO this is a "feature" that is better implemented in a custom theme, not in our development theme - it is not something that should be enforced universally. 

### TLDR;
IMO we should remove this styling from our development theme - thoughts?

**Comment by quatlin** [comment]
I agree, I don't see any reason to keep it.

**Comment by philcon93** [comment]
sounds aight, PR that
